### PR TITLE
popcorn-time: Add version 0.4.9

### LINKED ISSUE #10160: [Request]: Popcorn Time
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Popcorn Time

### Description

Popcorn Time is a multi-platform, free software BitTorrent client that includes an integrated media player ( Windows / Mac / Linux )

### Homepage

https://github.com/popcorn-official/popcorn-desktop

### Download Link(s)

https://github.com/popcorn-official/popcorn-desktop/releases

### Some Indication of Popularity/Repute

The official repo -> https://github.com/popcorn-official/popcorn-desktop has 7.7k stars and 526 forks

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #10160

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/popcorn-time.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.4.9",
+    "homepage": "https://github.com/popcorn-official/popcorn-desktop",
```
Comment (HUMORCE) [inline bucket/popcorn-time.json:3]:
```suggestion
    "homepage": "https://popcorntime.app",
```

File: bucket/popcorn-time.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    "shortcuts": [
+        [
+            "Popcorn-Time.exe",
+            "Popcorn Time",
+            "",
+            "src\\app\\images\\popcorntime.ico"
+        ]
+    ],
+    "checkver": "github",
```
Comment (HUMORCE) [inline bucket/popcorn-time.json:24]:
```suggestion
    "checkver": {
        "github": "https://github.com/popcorn-official/popcorn-desktop"
    },
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]

File: bucket/popcorn-time.json (hunk 1 above)
Comment (Zliced13) [reply bucket/popcorn-time.json:3]: The website doesn't work.

Comment (Zliced13) [reply bucket/popcorn-time.json:3]: I will do it anyway.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**